src/ontology/mondo-edit.obo

```diff
@@ -464918,6 +464920,2857 @@ is_a: MONDO:0005550 {source="https://orcid.org/0000-0002-5002-8648"} ! infectiou
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0019979 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog renal hypoplasia, unilateral
+
+[Term]
+id: MONDO:1010266
+name: renal hypoplasia, bilateral, non-human animal
+def: "Renal hypoplasia, bilateral that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:000858 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:000858"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0019980 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog renal hypoplasia, bilateral
+
+[Term]
+id: MONDO:1010267
+name: urolithiasis, non-human animal
+def: "Urolithiasis that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001033 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001033"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0024647 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog urolithiasis
+
+[Term]
+id: MONDO:1010268
+name: renal dysplasia, non-human animal
+def: "Renal dysplasia that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001135 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001135"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0019638 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog renal dysplasia
+
+[Term]
+id: MONDO:1010269
+name: hypospadias, non-human animal
+def: "Hypospadias that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001187 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001187"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0005345 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog hypospadias
+
+[Term]
+id: MONDO:1010270
+name: alkaptonuria, non-human animal
+def: "Alkaptonuria that occurs in non-human animals." [MONDO:patterns/nonhuman_disease]
+xref: OMIA:001243 {source="MONDO:equivalentTo"}
+is_a: MONDO:0005583 {source="OMIA:001243"} ! non-human animal disease
+intersection_of: MONDO:0005583 {source="https://orcid.org/0000-0002-5002-8648"} ! non-human animal disease
+intersection_of: MONDO:0700097 MONDO:0008753 {source="https://orcid.org/0000-0002-5002-8648"} ! cross-species analog alkaptonuria
+
+[Term]
+id: MONDO:1010271
+name: primary hyperoxaluria type I (Oxalosis I), non-human animal
```

Comment:
In human, "disease ~ type x" is based on the gene affected. Should we have only the species-specific term(s) instead of this generic term based on "general species unspecific" gene? 
Note that these species-specific diseases (based on species-specific genes) could be analogous to the human type x disease, but it is difficult to create an axiom referring to the general "species-not- specific" gene.